contracts/bond/ERC20SingleCollateralBond.sol

```diff
@@ -357,6 +357,13 @@ contract ERC20SingleCollateralBond is
         return address(_collateralTokens);
     }
 
+    /**
+     * @notice The ERC20 contract being used as collateral.
+     */
+    function collateralTokensSymbol() external view returns (string memory) {
```

Comment:
bit confused by the ticket - isnt the collateralToken exposed in the getter above this one?

Reply:
🤔 True ...there's not much point in wrapping the call to `symbol()`, as you can get the collateral tokens address & query that directly.

Happy to not have it 👍